crates/db_schema/src/utils.rs

```diff
@@ -368,6 +371,9 @@ fn run_migrations(db_url: &str) -> Result<(), LemmyError> {
   conn
     .run_pending_migrations(MIGRATIONS)
     .map_err(|e| anyhow::anyhow!("Couldn't run DB Migrations: {e}"))?;
+  conn
+    .batch_execute(include_str!("../../../replaceable_schema.sql"))
+    .with_context(|| "Couldn't run replaceable_schema.sql")?;
```

Comment:
I think this will break if you build a binary and execute it in a different folder. Probably need to use `include_str`()`

Reply:
Oops I must be blind.